src/scripts/main.js

```diff
@@ -1,3 +1,37 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+function complete(result) {
+  const div = document.createElement('div');
+
+  div.textContent = 'Promise was resolved!';
+  div.className = 'message';
+  document.body.appendChild(div);
+}
+
+function error(data) {
+  const div = document.createElement('div');
+
+  div.textContent = 'Promise was rejected!';
+  div.className = 'error-message';
+  document.body.appendChild(div);
+}
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve();
+  });
```

Comment:
you can write it more simply if you want  
```suggestion
  logo.addEventListener('click', resolve);
```